src/public/modules/forms/helpers/CsvMergedHeadersGenerator.ts

```diff
@@ -28,25 +39,32 @@ class CsvMergedHeadersGenerator extends CsvGenerator {
   /**
    * Returns current length of CSV file excluding header and meta-data
    */
-  length() {
+  length(): number {
     return this.unprocessed.length
   }
 
   /**
-   *
-   * @param {Object} decryptedContent
-   * @param {DisplayedResponse[]} decryptedContent.record
-   * @param {string} decryptedContent.created
-   * @param {string} decryptedContent.submissionId
+   * Adds an UnprocessedRecord to this.unprocessed
+   * @param decryptedContent
+   * @param decryptedContent.record
+   * @param decryptedContent.created
+   * @param decryptedContent.submissionId
    */
-  addRecord({ record, created, submissionId }) {
+  addRecord({
+    record,
+    created,
+    submissionId,
+  }: {
+    record: DisplayedResponseWithoutAnswer[]
+    created: string
+    submissionId: string
```

Comment:
Nope. The `record` in `UnprocessedRecord` is of another type: `{ [fieldId: string]: Response }`

Reply:
Ah ok! Wdyt about making this a type alias for readability

Reply:
I would prefer not to haha :) since this type is only referenced once

Reply:
this should be a type. even though its only used once in this file, functions calling this may need the type.

for example, i see that in the current code this type is actually a CsvRecord.submissionData from the decryption worker (which has not been translated to Typescript yet),

Ideally, the type would be exported from the decryption worker and imported and used here, but since that does not exist yet we should type it here.